chapters/mime-type/index.rst

```diff
@@ -1,4 +1,4 @@
-MIME Types in Jina
+MIME Types
```

Comment:
I don't understand the difference between this page and Primitive Data Types. Additionally should this not all fall under input/ouput?

Reply:
Let's fix that issue later. I don't want to rewrite or merge any docs right now. Just organize and standardize what we've got.